### PR TITLE
NO-TICK: Improve logging in the linear chain sync component.

### PR DESCRIPTION
**Pull request by goral09**

### CODE REVIEW

File: node/src/components/linear_chain_sync.rs [hunk 1]
```diff
@@ -570,7 +575,7 @@ where
             Event::BlockHandled(header) => {
                 let block_height = header.height();
                 let block_hash = header.hash();
-                trace!(?block_height, ?block_hash, "Block handled.");
+                trace!(%block_height, %block_hash, "nlock handled.");
```
Comment (afck) [inline node/src/components/linear_chain_sync.rs:578]:
```suggestion
                trace!(%block_height, %block_hash, "block handled.");
```

**Review by afck — APPROVED** [review]
Looks good; just one typo.

**Comment by goral09** [comment]
bors merge

**Pull request merged**